action.sh

```diff
@@ -9,4 +9,4 @@ INPUT_TOKEN=${INPUT_TOKEN:-"${{ secrets.GITHUB_TOKEN }}"}
 git config user.email ${INPUT_EMAIL}
 git config user.name ${INPUT_NAME}
 git config user.password ${INPUT_TOKEN}
-echo "::set-env name=GIT_USER::${INPUT_ACTOR}:${INPUT_TOKEN}"
+echo "name=GIT_USER::${INPUT_ACTOR}:${INPUT_TOKEN}" >> $GITHUB_ENV
```

Comment:
I think for both of these changes you should need `echo "GIT_USER=${INPUT_ACTOR}:${INPUT_TOKEN}" >> $GITHUB_ENV` instead

Reply:
You are right. It was an artifact from set-env's syntax.